prototype-kit/app/views/development-plan-event.html

```diff
@@ -0,0 +1,48 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+Question page template – {{ serviceName }} – GOV.UK Prototype Kit
+{% endblock %}
+
+{% block beforeContent %}
+{{ govukBackLink({
+  text: "Back",
+  href: "javascript:window.history.back()"
+}) }}
+{% endblock %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-l">
+
+    </h1>
+
+    <form class="form" action="/start-date" method="post">
```

Comment:
I presume a lot of this code is generated for us, but is this hard coded action in the form the common way to state the form flow?

Feels slighty odd that each page dictates what the next page is rather than having something that sits above them. It's not great for trying to see the full flow of the form and in my opinion a page shouldnt have to care about what the next page is.

Reply:
Yeah this is how the gov prototype kit tutorials etc do it, super clunky